src/mpi/datatype/type_indexed.c

```diff
@@ -143,6 +143,8 @@ int MPIR_Type_indexed(int count,
 
         new_dtp->has_sticky_lb = old_dtp->has_sticky_lb;
         new_dtp->has_sticky_ub = old_dtp->has_sticky_ub;
+
+        new_dtp->alignsize = old_dtp->alignsize;
```

Comment:
shouldn't be this a separate PR

Reply:
It is.  That got merged in.  I need to rebase on to `mpich/master`.